pkg/minikube/config/config.go

```diff
@@ -33,7 +33,7 @@ const (
 	WantReportError           = "WantReportError"
 	WantReportErrorPrompt     = "WantReportErrorPrompt"
 	WantKubectlDownloadMsg    = "WantKubectlDownloadMsg"
-	MachineName               = "name"
+	MachineName               = "profile"
```

Comment:
this is a little weird, maybe change the const to `Profile = "profile"` and then use that in GetMachineName?